docs/apm/correlations.asciidoc

```diff
@@ -2,17 +2,15 @@
 [[correlations]]
 === Find latency and error correlations
```

Comment:
This feature in 7.14 is specifically for `transaction latency` (as opposed to spans)